conseil-common/src/main/scala/tech/cryptonomic/conseil/common/tezos/Tables.scala

```diff
@@ -1751,6 +1757,9 @@ trait Tables {
 
     /** Index over (level) (database name endorsing_rights_level_idx) */
     val index2 = index("endorsing_rights_level_idx", level)
+
+    /** Index over (delegate) (database name ix_delegate_slot) */
+    val index3 = index("ix_delegate_slot", delegate)
```

Comment:
the generated index name referred in the val doesn't match with the db-schema which is `ix_delegate`

Reply:
the comment in the previous line should be amended accordingly, too